test/mocha/10-api.js

```diff
@@ -0,0 +1,146 @@
+/*!
+ * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const bedrock = require('bedrock');
+const {config} = bedrock;
+// apisauce is a wrapper around axios that provides improved error handling
+const {create} = require('apisauce');
+const https = require('https');
+const helpers = require('../helpers');
+const mockData = require('../mock.data');
+
+let accounts;
+let api;
+
+const baseURL =
+ `https://${config.server.host}${config['account-http'].routes.basePath}`;
+
+// simple quick func to check validation errors
+// TODO extend mocha should with this
+function validationError(result, errorMethod, expectedStatus = 400) {
+  result.status.should.equal(expectedStatus);
+  result.data.should.have.property('message');
+  result.data.should.have.property('type');
```

Comment:
At minimum I think we want to make assertion about `type` value.

Reply:
ok cool doing it. Also we could make this a custom should method on chai, but that might take some time to do (I have done custom expect, but not should).